src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyController.cs

```diff
@@ -137,6 +137,49 @@ public async Task<ViewResult> ChooseOrganisation()
         {
             return View(new ChooseOrganisationViewModel());
         }
+
+        [HttpGet]
+        [Route("need-bank-details")]
```

Comment:
Are these routes within the context of an organisation (account legal entity)?  Should that be in the route?  Or is it always at the account level?

Reply:
Good spot, I think bank details would be stored against a legal entity, not an account

Reply:
Although, this is in effect a new journey for the user as they've previously confirmed all their apprenticeships, this is a landing page for the second part of the (external) journey

Reply:
Should it be in it's own controller?

Reply:
Moved to its own controller in https://github.com/SkillsFundingAgency/das-employer-incentives-web/pull/23